### PR TITLE
run: set systmd-nspawn log level to error

### PR DESCRIPTION
**Pull request by dgonyeo**
Fixes https://github.com/appc/acbuild/issues/110. Probably. I can't reproduce the warnings on my system.

### CODE REVIEW

File: util/exec.go [hunk 1]
```diff
@@ -21,10 +21,10 @@ import (
 
 // Exec will execute the given binary with the given args, sending all messages
```
Comment (jonboulle) [inline util/exec.go]: false

Comment (jonboulle) [inline util/exec.go]: This helper seems questionable :/

Comment (jonboulle) [inline util/exec.go]: Questionably necessary that is

Comment (dgonyeo) [inline util/exec.go]: It saves 4 lines in 3 places, so it's not the best helper ever. Want me to nuke it?

Comment (jonboulle) [inline util/exec.go]: Please.

Comment (dgonyeo) [inline util/exec.go]: Done

**Comment by jonboulle** [comment]
LGTM, but you have problems spelling systmd apparently

**Comment by dgonyeo** [comment]
Good thing you didn't hire me for my spelling.

**Comment by jonboulle** [comment]
wait, I didn't hire you at all....

**Pull request merged**